static-site/src/sections/sars-cov-2-page.jsx

```diff
@@ -132,6 +133,21 @@ class Index extends React.Component {
       catalogueDatasets: sarscov2Catalogue.datasets,
       filterParsed: false
     };
+    if (this.props.uri === "/sars-cov-2") {
```

Comment:
[notes from zoom call yesterday] Let's remove the URL-based filtering for the time being, so this added code can be removed, and thus we only need `nonExistentDatasetName = this.props["*"]` set as component state here in the constructor.